crates/ruma-events/src/kinds.rs

```diff
@@ -414,6 +554,27 @@ pub struct RedactedStateEvent<C: RedactedStateEventContent> {
     pub unsigned: RedactedUnsigned,
 }
 
+impl<C: RedactedStateEventContent> JsonCastable<RedactedSyncStateEvent<C>>
```

Comment:
An implementation that is missing because there is no bound between `Redacted*` and `PossiblyRedacted*` is:

```rust
impl<C: StaticStateEventContent + RedactContent> JsonCastable<StrippedStateEvent<C::PossiblyRedacted>>
    for RedactedStateEvent<C::Redacted>
where
    C::Redacted: RedactedStateEventContent,
    C::PossiblyRedacted: PossiblyRedactedStateEventContent,
{
}
```